### ISSUE TITLE
Regarding BCI2008 Result

### ISSUE THREAD

**Issue by Romio123-max** [Altaheri/EEG-ATCNet#24]
I trained the model using your TensorFlow implementation and got results similar to those reported in the paper. However, when I used the PyTorch version of the same model from Braincode with identical training parameters, the accuracy dropped to around 75%. Can you explain why this might be happening?

**Comment by Altaheri** [comment]
Hi @Romio123-max, Thank you for your feedback and for testing both versions of the model.

We haven't yet evaluated the implementation in Braindecode, so we can’t confirm its alignment with our original results. However, we do have a PyTorch implementation on our side that yields results comparable to the TensorFlow version. We plan to release this PyTorch version soon as part of a comparative study for a new method.

We will review the Braindecode implementation to see if there’s anything that needs to be updated or clarified. We appreciate you bringing this to our attention!
